CHANGELOG-UNRELEASED.md

```diff
@@ -6,12 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Option to show NPM output when pulling deps during `hc test` [PR#1401](https://github.com/holochain/holochain-rust/pull/1401)
-- Adds scaffolding/skeleton for a future WASM conductor [#894](https://github.com/holochain/holochain-rust/pull/894)
```

Comment:
These lines must stay in! Changelogs from other PRs that got merged before!

Reply:
I'll get it right one day :man_facepalming: